win32/ctags_vs2013.vcxproj.filters

```diff
@@ -660,4 +663,4 @@
       <Filter>Header Files</Filter>
     </ClInclude>
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>

```

Comment:
This change for the end of file should not be added. git add -p may help you.

Reply:
Oops. That's my editor being too smart for its own good. I'll fix it.